src/templates/qa_hpwh.mdx

```diff
@@ -51,221 +50,514 @@
+      discharge pipe and pull and rotate the handle to open the valve for a few seconds to
+      ensure it opens fully. When performing this check, stand back and wear closed-toe shoes
+      to protect your feet from hot water. Once you begin to rotate the handle back close it
+      should snap close by itself. The discharge pipe should stop dripping shortly. If it
+      continues to drip, call a plumber to have the pressure relief valve replaced.
+      2. Check the expansion tank. Remove the cap and briefly press in the Schrader valve to
+      check that air (and not water) is expelled. If water is expelled the thermal expansion
+      tank has failed and will need replacement, call a plumber to have the thermal expansion
+      tank replaced. If air is expelled, the thermal expansion tank is working as expected.
+      The pre-charge pressure of the thermal expansion tank can be checked by turning off the
+      cold water main, opening a faucet to relieve the water pressure in the pipes, then
+      checking the pre-charge with a tire pressure gauge. Adjust as needed using a tire pump
+      to match the pressure written on the tank. Reinstall the cap on the Schrader valve if
+      one is present, and reopen the cold water main shutoff valve.
+    </Collapsible>
+
+  </Tab>
+  <Tab eventKey="report" style={{paddingTop:"1rem"}} title="Final Report">
+    <PrintSection label="Print Report">
+    ---
+    **Heat Pump Water Heater Quality Installation Documentation Report**
+
+    ##### Builiding Location
+    **Street Address:** {props.doc.location?.street_address}
+    **City:** {props.doc.location?.city}
+    **State:** {props.doc.location?.state}
+    **Zip:** {props.doc.location?.zip_code}
+
+
+    **Technician Name:** {props.doc.technician_name}
+
+    **Installation Company:** {props.doc.installation_company}
+
+    **Date:** {props.doc.install_date}
+
+    <Photo id="house_number_photo" label="House Number – Photo" required />
+
+    Report for {props.doc.customer_name}, and energy efficiency program quality assurance and quality control implementation contractor staff.
+
+    *The photos and information in this report use timestamps and geotags to prevent the reuse of photos from an unrelated project. The report's content is available only to the parties above. Any additional use such as for marketing and training purposes requires written permissions from the customer, in which case all timestamps, geotags, and other metadata from the photos and any identifiable characteristics that links the photos to this specific house will be removed prior. The homeowner / landlord is encouraged to retain a copy of this report which could help add value to a home sale and foster buyer confidence in the quality of the energy efficiency upgrade work that was performed. This report's data is protected by the standard data security and privacy best practices.*
+
+    <br/><br/>
```

Comment:
I'm not sure what purpose the `<br/>`s are serving, but we should not be trying to force the layout in this way from the template. We _could_ create a new component `<PageBreak>` that has its styling set to page break before it.

You can leave it this way for now, if needed. I have created a new issue for adding a `<PageBreak>` component.